lisa/wlgen/rta.py

```diff
@@ -44,15 +44,18 @@ class RTA(Workload):
 
     sched_policies = ['OTHER', 'FIFO', 'RR', 'DEADLINE']
 
-    def __init__(self, target, name, res_dir=None, json_file=None):
+    default_ftrace = 'main,task,loop,event,stats'
```

Comment:
looks like a list or a set would be more appropriate which is easier to manipulate in the python world. It can be trivially turned back into that comma-separate list using `','.join(foobar_list)` just before passing to rtapp.

Reply:
This is now done in a different way.